packages/ui/src/message-pane-input/MessagePaneInput.jsx

```diff
@@ -293,7 +302,7 @@ const MessagePaneInput = ({ onSendMessage, users, onAttachFile }) => {
             editorState={editorState}
             setEditorState={setEditorState}
             emojiSelect={<EmojiSelect />}
-            sendMessageHandler={sendMessage}
+            // sendMessageHandler={sendMessage}
```

Comment:
does this not affect the ability to send a message?
remove the sendmessage if it's not being used